### ISSUE TITLE
Client error response 404 Not Found url when create a message

### ISSUE THREAD

**Issue by dgpro** [rackspace/php-opencloud#661]
Hi,
I'm using rackspace/php-opencloud 1.16 on PHP Version 5.5.9-1ubuntu4.14, cURL 7.35.0.
I copied a code from here [add-message.php](https://raw.githubusercontent.com/rackspace/php-opencloud/master/samples/Queues/add-message.php) but when I try to create a message in the queue I get this error:

> Guzzle\Http\Exception\ClientErrorResponseException: Client error response [status code] 404 [reason phrase] Not Found [url] https://iad.queues.api.rackspacecloud.com/v1/[removed]/queues/test_queue in /vendor/guzzle/guzzle/src/Guzzle/Http/Exception/BadResponseException.php on line 43

Here is my code: [index.php](https://gist.github.com/dgpro/a4dffa4ca7dd59110fe4#file-index-php) (username and apiKey were removed)
and full error message: [error.html](https://gist.github.com/dgpro/a4dffa4ca7dd59110fe4#file-xerror-html)

**Comment by jamiehannaford** [comment]
@dgpro Are you 100% sure a queue exists with that name? If you're unsure you can use the [`hasQueue`](https://github.com/rackspace/php-opencloud/blob/working/lib/OpenCloud/Queues/Service.php#L184) method to check, like so:

``` php
$name = 'foo';

if (!$service->hasQueue($name)) {
   $service->createQueue($name);
}

$queue = $service->getQueue($name);
```

**Comment by dgpro** [comment]
Indeed.
I created a queue manually but it was failing for another reason, but when I copied that script over the queue was already deleted.
And the reason it was failing with [status code] 400 [reason phrase] Bad Request is that 'ttl' value was of type string (as I keept in my config), I've spent hours trying to find out that 'ttl' must be an integer which is frustrating.

By the way you don't need to get queue again if it has been created

```
$name = 'foo';
if (!$service->hasQueue($name)) {
   $queue = $service->createQueue($name);
} else {
   $queue = $service->getQueue($name);
}
```

Thanks for your help.
